x-pack/plugins/encrypted_saved_objects/server/saved_objects/encrypted_saved_objects_client_wrapper.ts

```diff
@@ -145,23 +151,26 @@ export class EncryptedSavedObjectsClientWrapper implements SavedObjectsClientCon
   }
 
   public async find(options: SavedObjectsFindOptions) {
-    return this.stripEncryptedAttributesFromBulkResponse(
-      await this.options.baseClient.find(options)
+    return await this.stripEncryptedAttributesFromBulkResponse(
```

Comment:
> The bulkGet already surfaces errors along these lines. However, the find operation does not. This complication can largely be overcome with enough effort.

Am I wrong or `find` returns the list of entities that have the same [SavedObject](https://github.com/elastic/kibana/blob/f29af1219d903a8dc5bfc2fd28058334f366c031/src/core/server/saved_objects/service/saved_objects_client.ts#L92) type that `bulkGet` uses and hence we can populate its [`error` property](https://github.com/elastic/kibana/blob/f29af1219d903a8dc5bfc2fd28058334f366c031/src/core/server/saved_objects/types.ts#L88) in the same way? Or you meant something else?

Also we'll need to analyze this `error` property and don't decrypt (just strip if attributes-to-encrypt are defined maybe) if it's defined (I missed in the initial implementation :see_no_evil: )...

Reply:
That's interesting that the types work out that way...  I don't believe we've ever returned saved-objects with the error property from the `find` operation before, but starting to do so now for encrypted saved-object attributes does seem tolerable. I was likely overreacting to the initial complications.

> Also we'll need to analyze this error property and don't decrypt (just strip if attributes-to-encrypt are defined maybe) if it's defined (I missed in the initial implementation 🙈 )..

Great point!